src/main.c

```diff
@@ -22,7 +22,11 @@ int main(int argc, char **argv)
         else die("Incorrect usage! Use\n\t--no-shell-escape\n\t--no-prompt");
     }
 
-    for(;;){
+    char *p;
+    int c,b,l,d;
```

Comment:
Variables should be declared right at the top of a function. A convention we follow throughout the code.